src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -3197,7 +3197,7 @@ exports[`DataTable custom theme 1`] = `
         class="StyledTable__StyledTableRow-sc-1m3u5g-2 "
       >
         <td
-          class="c3"
+          class="c3 "
```

Comment:
Can we try to understand the root cause of this extra space and clean it up?

Reply:
The select column changed to use `StyledDataTableCell` like the other header cells rather than just `TableCell`. If you look at line 3204 which is the first `column` you see it has the same space after the class name so I'm guessing its how `StyledDataTableCell` converts the pin values into CSS (and pre-existing.) I'll poke at it a little but it doesn't seem like it matters.

That does raise this question though:
Currently/pre-existing the select header cell isn't affected by theme.dataTable.header.extend like other header cells.
Should it be?

Reply:
It should be pulling stuff from the theme.

Reply:
The extra space on 3200 above (and all other existing variations on TableCell) is because TableCell passes along a className prop to StyledTableCell which ends up causing this space when className is undefined. Doesn't seem worth messing with.

Reply:
The select header cell now pays attention to theme.dataTable.header.extend like other header cells